docs/demos/lcore/weak_points_for_ai.html

```
@@ -0,0 +1,36 @@
+<!doctype html>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)